docs/Exp04-Tutor.md

```diff
@@ -56,17 +56,96 @@ Now, every time you open a new terminal, you can initialize the environment of B
 BLIS_INIT
 ```
 
-### 2.3. Intel CPU
+### 2.3. MKL-Intel CPU
 
-To do.
+#### 2.3.1 Intel oneAPI Base Toolkit
+
+If you have an Intel CPU, you should install **MKL** for BLAS.
+
+It's included by **Intel oneAPI Base Toolkit**. So we'll install the toolkit. Here is the official download [page](https://www.intel.cn/content/www/cn/zh/developer/tools/oneapi/base-toolkit-download.html).
+
+Choose the `Offline/Online Installer` for any system. If your **Package Manager**'s package type is supported, you can choose it instead.
+
+The website and installer (if you choose installer) will guide you, just follow them.
```

Comment:
Please provide the install commands without GUI guidance (which I believe should at least contain the `--silence` argument).